src/core/components/Search/SearchResults.js

```diff
@@ -31,8 +31,10 @@ export default class SearchResults extends React.Component {
 
     let searchResults;
     let messageText;
+    let hideMessageText = false;
 
     if (query && count > 0) {
+      hideMessageText = true;
```

Comment:
It's a nice bit of additional data which could be useful - seems a shame if it's limited to just screenreader users in a way.